fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/domain/search/SavingsTransactionSearch.java

```diff
@@ -43,7 +43,7 @@ public static class Filters {
     @Data
     public static class RangeFilter<T> {
 
-        private RangeOperator operator;
+        private SqlOperator operator;
```

Comment:
Now this I don't like. We're exposing "SqlOperators" to higher layers (i.e.API) which shouldn't happen.